src/Toggle/Toggle.js

```diff
@@ -120,6 +120,10 @@ class Toggle extends Component {
      */
     thumbStyle: PropTypes.object,
     /**
+    * Override the inline styles for thumb when the toggle switch is toggled on.
+    */
+    thumbWhenSwitchedStyle: PropTypes.object,
```

Comment:
@oliviertassinari if it's going to be a breaking change for any existing props we should get it in `0.15.0`.